src/test/java/hec/DatabaseVersioning.java

```diff
@@ -0,0 +1,42 @@
+package hec;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+import java.io.*;
+import java.nio.file.*;
+import java.util.ArrayList;
+
+
+
+public class DatabaseVersioning{
+
+    @Test
+    public void Original_db_can_update_to_newest() throws Exception{        
+        File testdb = get_test_file("/database/ResSimTest_20200101.db");                
+        try( TimeSeriesDatabase db = new JdbcTimeSeriesDatabase(testdb.getCanonicalPath(), JdbcTimeSeriesDatabase.CREATION_MODE.OPEN_EXISTING_UPDATE);){
+            assertEquals( get_latest_version(db.getVersions()), db.getVersion() );
+        }
+        
+        testdb.delete();
+    }
+
+
+
+    public File get_test_file( String test_database_resource ) throws Exception{
```

Comment:
can we make get_test_file  private, and the next.

Reply:
yes, that shouldn't be an issue.